### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by xerta555**

### CODE REVIEW

File: packages/marketplace/src/resources/locales/fr.json [hunk 1]
```diff
@@ -42,6 +45,17 @@
       "reloadNow": "Recharger maintenant",
       "reloadLater": "Recharger plus tard"
     },
+    "backupModal": {
+      "title": "Importer/Exporter",
```
Comment (theRealPadster) [inline packages/marketplace/src/resources/locales/fr.json:49]: The title in English is "Back up/Restore"

**Review by theRealPadster — CHANGES_REQUESTED** [review]

File: packages/marketplace/src/resources/locales/fr.json (hunk 1 above)
Comment (xerta555) [reply packages/marketplace/src/resources/locales/fr.json:49]: Strange, i see this in the en file
image

Comment (kyrie25) [reply packages/marketplace/src/resources/locales/fr.json]: > Strange, i see this in the en file image

That would be this, which is also `Back up/Restore`:
image

Comment (kyrie25) [reply packages/marketplace/src/resources/locales/fr.json]: You can reuse strings btw, in case you haven't noticed.

Comment (xerta555) [reply packages/marketplace/src/resources/locales/fr.json]: Ok done.

**Review by theRealPadster — APPROVED** [review]

**Comment by theRealPadster** [comment]
Thanks!

**Pull request merged**